### PR TITLE
add HOST env var to cms app creator

### PR DESCRIPTION
**Pull request by thescubageek**
@mmitchellg5 @johnlucia I'm trying to set it up so that all new CMS's come with an ENV var named HOST that is set to the full URL to the CMS itself, eg for g5-cms-6d4u4sc-madrona:

HOST=https://g5-cms-6d4u4sc-madrona.herokuapp.com

Can you guys help me figure out if this is the right path and how we go about testing this?

### CODE REVIEW

File: app/models/client_app.rb [hunk 1]
```diff
@@ -67,7 +67,7 @@ def heroku_app
   end
 
   def host
-    "#{name}.herokuapp.com"
```
Comment (thescubageek) [inline app/models/client_app.rb]: This method wasn't being used by anything, so I hijacked it for my own nefarious means.

**Comment by johnlucia** [comment]
I think you can just use the heroku app name formatter gem and your testing is pretty much already taken care of. Would go something like this...

```formatter = G5HerokuAppNameFormatter::Formatter.new(client.urn, ["cms"])

formatter.cms_url```

https://github.com/G5/g5_heroku_app_name_formatter

**Comment by variousred** [comment]
+1 to @johnlucia comment. What is this going to be used for?

**Comment by thescubageek** [comment]
Agree with @johnlucia that approach makes much more sense. Reworking the code in the cms-consolidation to not require this env var.

**Pull request closed without merging**